packages/app/src/app/store/modules/editor/index.js

```diff
@@ -43,18 +32,27 @@ export default Module({
       y: 0,
       content: 'browser',
     },
-  },
-  getters: {
-    isAllModulesSynced,
-    currentSandbox,
-    currentModule,
-    mainModule,
-    currentPackageJSON,
-    currentPackageJSONCode,
-    parsedConfigurations,
-  },
-  computed: {
-    isModuleSynced,
+    get isAllModulesSynced() {
```

Comment:
Will this allow you to also set a `set`?

Reply:
Yeah, we can allow for this. You mean doing a:

```js
set isAllModulesSynced () {
  this.changedModuleIds = []
}
```

Type of thing?

Reply:
Let me try to get this running, it is nice thing to have :)

Reply:
Oh, haha, this actually already works :) It requires a getter set... but yeah, of course you would have a getter first :)

Reply:
Awesome!